assets/sass/components/idea-hub/_googlesitekit-idea-hub-dashboard-ideas-widget.scss

```diff
@@ -126,11 +126,14 @@
 		vertical-align: top;
 	}
 
-	.googlesitekit-idea-hub__idea--text {
-		color: $c-idea-hub-name;
-		font-size: 0.875rem;
-		line-height: 1.45;
-		margin: 0;
+	p {
+
+		&.googlesitekit-idea-hub__idea--text {
```

Comment:
How about we leave this selector as-is and change [the component](https://github.com/google/site-kit-wp/blob/1bc57c4c6d14779952f356746999a75cfbb76e12/assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/Idea.js#L154) to use a `div` instead? I think the only time we really want to use `p` is in the WP dashboard or admin notice where we want to inherit wp-admin styles.

Reply:
I'am not too keen on using a `div` from a semantic point of views since we are displaying some text. 😬